### ISSUE TITLE
Camptix Notify: "is not" filter is not working correctly

### ISSUE THREAD

**Issue by timiwahalahti** [WordPress/wordcamp.org#500]
It's not possible to segment email for attendees that have not answered specific question while registration. The "is not" filter only takes into account the question if attendee has some kind of answer in it.

For example: we have non-mandatory question with two radio options "Yes" and "No".  Some attendees haven't selected option and the answer is left empty, in summarize these are shown as "None" answers. At some point, organizers make the field mandatory and do want to send an email to every attendee with "No" answer and without any answer. If you set the filter to "is not yes", notify counts only the attendees with "No" answer and doesn't detect attendees without an answer at all.

**Expected behavior**

When using "is not" filter, attendees without an answer should be taken into account. Or there should be another way to send emails to attendees without an answer.

**WordCamp**

WCEU 2020

Happy to provide more information if needed :)

**Comment by ryelle** [comment]
Good catch— would a new condition called "is empty" work? That way it can be used for all question types, not just yes/no. You could filter by `[question] is no or [question] is empty`

Edit: I do think it makes sense to keep "is not" matching only answers, since it's "people who answered this question with not yes".

**Comment by timiwahalahti** [comment]
> would a new condition called "is empty" work?

👍
